Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -257,14 +268,81 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
 
                 for (int i = 0; i < numColliders; i++)
                 {
-                    grabbable = queryBuffer[i].GetComponent<NearInteractionGrabbable>();
+                    Collider collider = queryBuffer[i];
+                    grabbable = collider.GetComponent<NearInteractionGrabbable>();
+                    // Additional check: is grabbable in the camera frustrum
                     if (grabbable != null)
```

Comment:
```c#
if (grabbable != null && !isInFOV(collider))
{
    grabbable = null;
}
```

Reply:
+1

Reply:
Whoops. I made this PR too quickly. Nice catch.